packages/app-builder/src/routes/_builder+/cases+/inboxes.$inboxId.tsx

```diff
@@ -69,50 +70,119 @@ export async function loader({ request, params }: LoaderFunctionArgs) {
 }
 
 export default function Cases() {
-  const {
-    casesData: { items: cases, ...pagination },
-    filters,
-  } = useLoaderData<typeof loader>();
+  const { casesData: initialCasesData, filters: initialFilters } =
+    useLoaderData<typeof loader>();
   const inboxId = useParam('inboxId');
   const { t } = useTranslation(casesI18n);
 
+  const [casesData, setCasesData] = useState(initialCasesData);
+  const { items: cases, ...pagination } = casesData;
+
+  const [filters, setFilters] = useState(initialFilters);
+  if (initialFilters !== filters) {
+    setFilters(initialFilters);
+    setCasesData(initialCasesData);
+  }
+
+  const {
+    state: paginationState,
+    next,
+    previous,
+    reset,
+  } = useCursorPagination();
+
+  const fetcher = useFetcher<typeof loader>();
   const navigate = useNavigate();
-  const navigateCasesList = useCallback(
-    (casesFilters: CasesFilters, pagination?: PaginationParams) => {
-      navigate(
+
+  useEffect(() => {
+    if (paginationState.isPristine) {
+      return;
+    }
+
+    fetcher.submit(
+      qs.stringify(
         {
-          pathname: getRoute('/cases/inboxes/:inboxId', {
-            inboxId: fromUUID(inboxId),
-          }),
-          search: qs.stringify(
-            {
-              statuses: casesFilters.statuses ?? [],
-              dateRange: casesFilters.dateRange
-                ? casesFilters.dateRange.type === 'static'
-                  ? {
-                      type: 'static',
-                      endDate: casesFilters.dateRange.endDate || null,
-                      startDate: casesFilters.dateRange.startDate || null,
-                    }
-                  : {
-                      type: 'dynamic',
-                      fromNow: casesFilters.dateRange.fromNow,
-                    }
-                : {},
-              offsetId: pagination?.offsetId || null,
-              next: pagination?.next || null,
-              previous: pagination?.previous || null,
```

Comment:
just double checking here: "previous" can now be removed from the backend API ?

Reply:
(if yes, let's also remove it on the openapi spec as an option)

Reply:
Yes, just discussed it with Pascal, next and previous params will be deleted from the input